packages/manager/core/vite-config/package.json

```diff
@@ -15,16 +15,16 @@
   "types": "./src/index.d.ts",
   "dependencies": {
     "@ovh-ux/manager-dev-server-config": "^6.1.3",
-    "@vitejs/plugin-legacy": "^5.4.3",
-    "@vitejs/plugin-react": "^4.3.4",
+    "@vitejs/plugin-react-swc": "^3.9.0",
     "express": "^4.17.1",
     "http-proxy-middleware": "^2.0.7",
     "terser": "^5.16.0",
     "vite-plugin-static-copy": "^2.0.0",
     "vite-plugin-svgr": "^3.2.0",
-    "yn": "^5.0.0"
+    "yn": "^5.0.0",
+    "vite": "^6.0.7"
```

Comment:
If "vite": "^6.0.7" is in `dependancies`. We can add it only in `devDependancies` ?